SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -46,6 +46,7 @@ class USpatialWorkerConnection;
 class USpatialWorkerFlags;
 
 DECLARE_DELEGATE(PostWorldWipeDelegate);
+DECLARE_DYNAMIC_DELEGATE(FShutdownEvent);
```

Comment:
Do we need a dynamic delegate here? This could work the same:
```suggestion
DECLARE_DELEGATE(FShutdownEvent);
```

Reply:
It could also be useful to make it multicast.

Reply:
I don't think we need a dynamic delegate here but I don't see the benefit of making it a multicast.

Reply:
You don't think dynamic is needed because the game instance life will outlive the net driver? Seems safer just to make it a dynamic delegate.
Making it multicast so multiple things can bind to the event does seem useful though

Reply:
I tend to always prefer regular delegates to dynamic ones, unless we need it in a blueprint function or property, since dynamic delegates' `AddDynamic` is essentially the same as regular delegates' `AddUObject`.
And making it dynamic was so that multiple things could react to this event; it also ensures no one overwrites your event with their own by accident.

Reply:
Why are we adding a delegate in the first place? The function we call from it (`USpatialGameInstance::DestroySpatialConnectionManager`) just calls destroy functions the connection manager and the global state manager (yay for naming :D). 
The SpatialNetDriver already has direct UPROPERTY pointers to both those objects, so why not just call those directly instead of going through the game instance?

Reply:
re: dynamic - okay adding as UObject seems sufficient also
re: why delegate at all - construction of these objects is done in the GameInstance, so for clarity it's best to keep the destruction of these objects there as well